src/fides/api/ctl/database/seed.py

```diff
@@ -385,3 +404,128 @@ async def load_default_resources(async_session: AsyncSession) -> None:
     await load_default_organization(async_session)
     await load_default_taxonomy(async_session)
     load_default_dsr_policies()
+
+
+async def load_samples(async_session: AsyncSession) -> None:
```

Comment:
This function is probably the grossest part. It effectively just repurposes other functions to take all the Pydantic models and then convert those into database objects. Some of that requires reformatting the data into formats that the various internal functions expect, etc. etc.

I tried to avoid calling any ORM creation methods directly, since I figured that'd be too easy to break.